src/data/javascript-quiz.ts

```diff
@@ -1970,7 +1984,8 @@ const javascriptQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/IIFE"
   },
   {
-    Question: "In JavaScript, which of the following is an example of a macrotask?",
+    Question:
+      "In JavaScript, which of the following is an example of a macrotask?",
```

Comment:
please remove this change here because you already included this change in your other PR

Reply:
Thank you for pointing it out, the changes were done.